modules/document_repository/ajax/categoryEdit.php

```diff
@@ -15,14 +15,14 @@
 if (!$user->hasPermission('document_repository_view')
     && !$user->hasPermission('document_repository_delete')
 ) {
-    header("HTTP/1.1 403 Forbidden");
+    header('HTTP/1.1 403 Forbidden');
```

Comment:
@intralizee to avoid missing important changes during review, we generally ask to hold off on esthetic chnages in the same PRs. i.e. the quotes could be made in a PR on their own which would be reviewed/tested with the knowledge that there should not be any behavioural changes